.github/issue_template.md

```diff
@@ -29,7 +29,7 @@ composer require --dev phpro/grumphp
 # Your actions
 # Please add the steps on how to reproduce the issue here.
 
-# Run GrumpHP:
+# Run GrumPHP:
 git add -A && git commit -m"Test"
```

Comment:
Reverted this one : `commit -am""` doesnt add unknown files to git + it requires the git EXEC inside the git hook to accept the diff through stdin.

Reply:
I never use the `commit -am""` I don't like it.
Thanks for the explanation.